packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -82,6 +83,12 @@ export interface DatepickerProps extends UIComponentProps, Partial<ICalendarStri
 
   /** Controls the calendar's open state. */
   calendarOpenState?: boolean;
+
+  /** Controls whether there is a date that is selected by default. */
```

Comment:
```suggestion
  /** Initial selectedDate value. */
```

This would be in my opinion closer to the description of the `default`s that we have in other components.